docs/baseplate/core.rst

```diff
@@ -58,6 +58,43 @@ integrations <integration/index>`.
 .. autoclass:: TraceInfo
    :members: from_upstream
 
+Edge Request Context
+--------------------
+
+If the application is directly receiving requests from a front-end clients or if it
+is receiving and wants to utilize the context data for that request, there are
+some support interfaces available for easily working with the request context data,
+passing context down, or picking up passed down contexts.
+
+If the application is an edge service that is directly serving reddit client
+requests, then the application should procure an
+:py:class:`~baseplate.core.AuthenticationContext` object and then store it along
+with the other context data in an :py:class:`~baseplate.core.EdgeRequestContext`
+object attached to the current context::
+
+    authentication_token = retrieve_token_from_authentication_service()
```

Comment:
should the following section about authentication token hydration be subsumed into this or otherwise updated?

Reply:
🤔 
Maybe it makes sense to deprecate attaching the authentication_context directly to the request object and just run everything through the `EdgeRequestContext`?